styles.css

```diff
@@ -0,0 +1,3 @@
+header {
+  color: blue;
+}
```

Comment:
You should give style to specific tag `<h1>`, if not the whole header will effect the text color of `blue`

eg: 
`header > h1 { color: blue; }`

Reply:
Can you please update this?
Beside this, all are well-structured 🎉